tiny_dnn/util/serialization_layer_list.h

```diff
@@ -44,6 +44,7 @@ void register_layers(T* h) {
   h->template register_layer<elu_layer>("elu");
   h->template register_layer<tanh_p1m2_layer>("tanh_scaled");
   h->template register_layer<softplus_layer>("softplus");
+  h->template register_layer<softplus_layer>("softsign");
```

Comment:
@nishnik
Thx to your contribution ;)
s/softplus/softsign/
This might be the cause of the error.